### DISCUSSION TITLE
Unable to update new rule in local.rules

### DISCUSSION THREAD

**Discussion by patelyogi95** [Security-Onion-Solutions/securityonion#9720] [Unsupported Versions]
Hi, I am trying to addnew rule. once i added new rule in local.rules and when i run salt-call  state.highstate command and so-rule-update command, i am not able to see new rule in local.rules or all.rules. 

here is the rule which i added : alert tcp any any -> $HOME_NET any (msg: "Security Onion under Attack!"; content: "Onionattack"; flow:to_server; nocase; sid:6786877; metadata:policy securityips; rev:1;)

**Comment by reyesj2** [comment]
Are you adding the rule in ```/opt/so/saltstack/local/salt/idstools/local.rules```?
